sippy-ng/src/component_readiness/CompReadyVars.js

```diff
@@ -240,6 +264,141 @@ export const CompReadyVarsProvider = ({ children }) => {
     setCapability(capabilityParam)
   }
 
+  const initialViews = {
+    Default: {
+      config: {
+        help: 'The most commonly used view and focuses on amd64',
+        'Group By': defaultGroupByCheckedItems,
+        'Exclude Arches': defaultExcludeArchesCheckedItems,
+        'Exclude Networks': defaultExcludeNetworksCheckedItems,
+        'Exclude Clouds': defaultExcludeCloudsCheckedItems,
+        'Exclude Upgrades': defaultExcludeUpgradesCheckedItems,
+        'Exclude Variants': defaultExcludeVariantsCheckedItems,
+        Confidence: defaultConfidenceParam,
+        Pity: defaultPityParam,
+        'Min Fail': defaultMinFailParam,
+        'Ignore Missing': defaultIgnoreMissingParam,
+        'Ignore Disruption': defaultIgnoreDisruptionParam,
+      },
+    },
+    Strict: {
+      config: {
+        help: 'Default view except Pity and MinFail set to 0',
```

Comment:
This is probably not to useful IMHO, you'll trigger statistically significant results on pass rate changes on the order of 0.01%. It'll turn most of the board red.